enhancements/api-review/custom-tags-aws.md

```diff
@@ -88,9 +88,10 @@ The userTags field is intended to be set at install time and is considered immut
 
 If the userTags field is changed post-install, there is no guarantee about how an in-cluster operator will respond to the change. Some operators may reconcile the change and change tags on the AWS resource. Some operators may ignore the change. However, if tags are removed from userTags, the tag will not be removed from the AWS resource.
 
-### Update design details
+### Updated design details
 
-The `experimentalPropagateUserTags` field is changed to `propagateUserTags` for GA version.
+A new field `propagateUserTags` is added in GA version. The `experimentalPropagateUserTags` field will be deprecated in future release versions.
+When both fields are set, `experimentalPropagateUserTags` takes precedence.
```

Comment:
we'd deprecate it now(when the ga field is added). we'd remove it in a future release.

Reply:
Can the flag used to propagate user flags `propagateUserTags` be removed and made a default propagation of user-defined tags ? Are there usecases that might require a switch like propagateUserTags . As similar switch needs to be considered for other cloud providers.

Reply:
if i understand your suggestion (default to just always propagate usertags when usertags are specified in the install config?) that feels like a breaking behavior change and also i'm not seeing the benefit of taking the choice away from admins.

Reply:
Thanks, the breaking behavior change perspective makes sense.

Reply:
`experimentalPropagateUserTags` is a boolean field in the install config--not a boolean pointer. Therefore it is impossible for `experimentalPropagateUserTags` to not be set, right? There is no difference between not setting the field, and setting the field to `false`.

Reply:
D'oh there is a whole section on converting this to a pointer. My mistake! re /lgtm